api/types/role.go

```diff
@@ -579,15 +579,15 @@ func (r *RoleV3) CheckAndSetDefaults() error {
 	}
 
 	if r.Spec.Allow.AppLabels == nil {
-		r.Spec.Allow.AppLabels = Labels{Wildcard: []string{Wildcard}}
+		r.Spec.Allow.AppLabels = Labels{}
 	}
 
 	if r.Spec.Allow.KubernetesLabels == nil {
-		r.Spec.Allow.KubernetesLabels = Labels{Wildcard: []string{Wildcard}}
+		r.Spec.Allow.KubernetesLabels = Labels{}
 	}
 
 	if r.Spec.Allow.DatabaseLabels == nil {
-		r.Spec.Allow.DatabaseLabels = Labels{Wildcard: []string{Wildcard}}
+		r.Spec.Allow.DatabaseLabels = Labels{}
 	}
```

Comment:
Remove these `if`s entirely...?

Also, shouldn't the `NodeLabels` wildcard above these be got rid of as well?

Reply:
I've now removed the node_labels wildcards as well.